api/KeyboardEvent.json

```diff
@@ -667,7 +667,7 @@
             "status": {
               "experimental": false,
               "standard_track": false,
```

Comment:
suggestion: since this is specified, let's reflect that in the `standard_track` value

```suggestion
              "standard_track": false,
```

Reply:
Don't see any changes here

Reply:
Sorry, copy and paste failure:

```suggestion
              "standard_track": true,
```